doc/mgr/nfs.rst

```diff
@@ -289,6 +289,15 @@ for permissible values.
 value is `no_root_squash`. See the `NFS-Ganesha Export Sample`_ for
 permissible values.
 
+``<sectype>`` specifies what kind of authentication methods will be used when
+connecting to the export. Valid values include "krb5p", "krb5i", "krb5", "sys",
+and "none". More than one value can be supplied. The server will negotatiate a
```

Comment:
It'd be great to note how more than one value may be supplied.  Multiple `--sectype foo --sectype bar` switches? Comma separated `--sectype foo,bar`, or ??  Maybe add an example invocation?

Reply:
both work, both are now documented, with examples.